SourceBuild.sln

```diff
@@ -0,0 +1,77 @@
+﻿
```

Comment:
Did you explore the Arcade way of doing this which involves the [DotNetBuildFromSource](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L13) and [ExcludeFromSourceBuild](https://github.com/dotnet/arcade/blob/cdd5d318667c5af1740d69c8a7755e23e7964aeb/src/Microsoft.DotNet.Arcade.Sdk/sdk/Sdk.targets#L5) properties?

Reply:
Yes, we have infrastructure for source build. No need to create a separate solution or build.cmd

Reply:
Thanks, working now on re-using our existing solutions and setting Exlcude where appropriate.